src/Sarif.Multitool.Library/FileWorkItemsCommand.cs

```diff
@@ -132,6 +132,7 @@ private bool ValidateOptions(FileWorkItemsOptions options, SarifWorkItemContext
             valid &= ValidateHostUri(options.HostUri, sarifWorkItemContext);
 
             valid &= options.ValidateOutputOptions();
+            valid &= options.ValidateOutputFormatOptions();
```

Comment:
>ValidateOutputFormatOptions [](start = 29, length = 27)

Can we find a way for users to be able to simply call a ValidateOptions method that pulls in everything?

Reply:
Combined into `Validate`.

---
In reply to: [500451538](https://github.com/microsoft/sarif-sdk/pull/2100#discussion_r500451538) [](ancestors = 500451538)